README.md

```diff
@@ -22,7 +22,32 @@ To get started with the quickstart application follow these steps. Steps 1-6 wil
 11. [Make client to client call](#bullet11)
 12. [Make client to PSTN call](#bullet12)
 
-### <a name="bullet1"></a>1. Install the TwilioVoice framework using Cocoapods
+### <a name="bullet1"></a>1. Install the TwilioVoice framework
+
+**Carthage**
+
+Add the following line to your Cartfile
+
+```
+github "twilio/twilio-voice-ios"
+```
+
+Then run `carthage bootstrap` (or `carthage update` if you are updating your SDKs)
+
+On your application targets’ “Build Phases” settings tab, click the “+” icon and choose “New Run Script Phase”. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
+
+```
+/usr/local/bin/carthage copy-frameworks
```

Comment:
do you need `/usr/local/bin` here? In other examples you reference just `carthage`

Reply:
Just `carthage` actually works fine. The path instruction is actually from the [Carthage README](https://github.com/Carthage/Carthage) and adopted by Video and Voice.

Reply:
Ah. Okay may have just been an oversight by them then. I'm fine with either. I was nitpicking.